tool/src/webapp/js/lessons-calendar.js

```diff
@@ -5,6 +5,8 @@ $(function(){
     $('.calendar-div').fullCalendar({
         displayEventTime: false,
         height: 400,
+        windowResize: function(view) {
+        },
```

Comment:
Calendar 'handleWindowResize' is set to true by default and I was testing using this method,  forgot to remove, my bad.thanks for pointing out. Have removed it and updated PR.